src/arrays/referencing_array_and_its_elements/hard/q5/StudentsNeedHelp.java

```diff
@@ -0,0 +1,38 @@
+package arrays.referencing_array_and_its_elements.hard.q5;
+/*
+Write a Java program that takes an array of 10 doubles between 0 and 100, indicating scores for 10 students, and then
```

Comment:
Change question text to:
`Write a Java program that takes an array of 10 doubles between 0 and 100, indicating scores for ten students. Then, traverse through the array and print out the index of students on the same line separated by spaces (Eg: "3 4 6 " ) who got below 60% in the course, indicating that those students need help. Your program should use a method called SNH that takes a double array and print the indices of the students needing help, and the method should be general enough to work with any array length. Assume that the user will enter numbers between 0 and 100 (inclusive).`